crates/ark/src/modules/rstudio/document-api.R

```diff
@@ -177,13 +177,15 @@ asRangeList <- function(location) {
 #' @export
 .rs.api.documentSaveAll <- function() {
     # This function excludes untitled files in RStudio:
-    invisible(.ps.ui.executeCommand("workbench.action.files.saveAllTitled"))
+    .ps.ui.executeCommand("workbench.action.files.saveAllTitled")
+    return(TRUE)
```

Comment:
Here and in line 190 are for consistency with what the rstudioapi functions return.